debian/pixelated-user-agent.postinst

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+export PATH=/bin:/usr/bin:/sbin:/usr/sbin
+
+systemctl restart pixelated-server
```

Comment:
Systemd is the default process manager in Debian, the only operating system supported by LEAP. I don't think this should be a concern.